packages/batch-submitter/src/batch-submitter/batch-submitter.ts

```diff
@@ -73,7 +73,11 @@ export abstract class BatchSubmitter {
       this.l2ChainId = await this._getL2ChainId()
     }
     await this._updateChainInfo()
-    await this._checkBalance()
+
+    if (!(await this._hasEnoughETHToCoverGasCosts())) {
+      await sleep(this.resubmissionTimeout)
```

Comment:
This will loop and throw a bunch of `fatal` logs. Is that OK?

Reply:
I think so yes — it is indeed fatal IMO. The service's purpose for existing is in a fatal state — it cannot submit batches

Reply:
This PR changes the functionality of the config option that sets the minimum safe ether balance. This is something that we have to be very mindful of because there is now a delta where it was possible to send batches before and its now not possible to send batches given the same config and the same amount of eth. This happens when the balance < min safe balance but balance > cost to submit a batch

Reply:
Keep in mind that we _also_ have another service running which sends us alerts when the amount if below a separately configured minimum safe balance amount.